.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -248,7 +260,19 @@ def ACCHostVerificationTest(String version, String build_type) {
                         vcvars64.bat x64 && \
                         cmake.exe ${WORKSPACE} -G Ninja -DBUILD_ENCLAVES=OFF -DHAS_QUOTE_PROVIDER=OFF -DCMAKE_BUILD_TYPE=${build_type} -DNUGET_PACKAGE_PATH=C:/oe_prereqs -Wdev && \
                         ninja -v && \
-                        ctest.exe -V -C ${build_type} -R host_verify --output-on-failure --timeout ${CTEST_TIMEOUT_SECONDS}
+                        ninja install && \
```

Comment:
From my understanding, this test is to verify the quotes created at the earlier steps. This is now modified to then also install the host verification package and install the samples and test.

I think this should stay separate and we should have a HostVerificationPackage testing phase instead of appending to the end of an existing (long-running) test.

Reply:
@BRMcLaren are you suggesting we wait until #3271 is resolved and we can pick a package from build?

Reply:
I'm suggesting make another test that specifically builds the package, installs it and the runs the samples. It would likely be better than tacking it on to the end of a current test.

The cpack issue you linked is unrelated to host verify, it's for the sdk package.

Reply:
Done with the new function, I would like both of you to review it again, please. Thanks!